File: PTJ1-Syntax-Translation-Warmup/starter/java/Main.java
import java.util.Arrays;

public class Main {
	static String greeting(String name) {
		// TODO: return "Hello, <name>!"
		return "";
	}

	static int absoluteValue(int value) {
		// TODO: return the non-negative version of value
		return 0;
	}

	static boolean isEven(int value) {
		// TODO: return true when value is even
		return false;
	}

	static String fizzBuzzLabel(int value) {
		// TODO: return Fizz, Buzz, FizzBuzz, or the number as text
		return "";
	}

	public static void main(String[] args) {
		int[] checks = { -7, -2, 0, 3, 5, 15 };
		System.out.println(greeting("Avery"));
		System.out.println(absoluteValue(-7));
		System.out.println(isEven(12));
		System.out.println(Arrays.toString(Arrays.stream(checks).mapToObj(Main::fizzBuzzLabel).toArray()));
	}
}

File: PTJ1-Syntax-Translation-Warmup/starter/cpp/main.cpp
#include <iostream>
#include <string>
#include <vector>

std::string greeting(const std::string& name) {
	// TODO: return "Hello, <name>!"
	return "";
}

int absoluteValue(int value) {
	// TODO: return the non-negative version of value
	return 0;
}

bool isEven(int value) {
	// TODO: return true when value is even
	return false;
}

std::string fizzBuzzLabel(int value) {
	// TODO: return Fizz, Buzz, FizzBuzz, or the number as text
	return "";
}

int main() {
	std::vector<int> checks = {-7, -2, 0, 3, 5, 15};
	std::cout << greeting("Avery") << "\n";
	std::cout << absoluteValue(-7) << "\n";
	std::cout << std::boolalpha << isEven(12) << "\n";
	for (const int value : checks)
		std::cout << fizzBuzzLabel(value) << " ";
	std::cout << "\n";
}
